packages/mds-stream/kafka/helpers.ts

```diff
@@ -0,0 +1,36 @@
+import Kafka, { ProducerStream } from 'node-rdkafka'
+import log from '@mds-core/mds-logger'
+import { ProducerOptions, StreamOptions, Producer } from './types'
+
+const {
+  env: { KAFKA_HOST = 'localhost:9092' }
+} = process
+
+export const defaultKafkaErrorHandler = async (err: any) => {
+  await log.error(`Kafka Error ${JSON.stringify(err)}`)
+}
+
+export const createWriteStreamWrapper = (
+  producerOptions: Partial<ProducerOptions>,
+  streamOptions: Partial<StreamOptions>,
+  errorHandler?: (err: any) => Promise<void>
+) => {
+  const stream = ((Kafka.Producer as unknown) as Producer).createWriteStream(
```

Comment:
If I were to hazard a guess I'd say it's the classic
> Conversion of type 'string' to type 'number' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
[See ts docs](https://microsoft.github.io/TypeScript-New-Handbook/everything/#type-assertions)
As to why `typeof Kafka.Producer !== Producer`, not sure.

Reply:
Yes, it's necessary due to what MaxJ mentioned. `Kafka.Producer !== Producer` right now due to a type issue within the kafka library we're using, which they've resolved at the tip of master but not released yet.